spyderlib/plugins/ipythonconsole.py

```diff
@@ -180,12 +176,22 @@ def setup_page(self):
 
         interface_layout = QVBoxLayout()
         interface_layout.addWidget(banner_box)
-        interface_layout.addWidget(gui_comp_box)
         interface_layout.addWidget(pager_box)
         interface_layout.addWidget(calltips_box)
         interface_layout.addWidget(ask_box)
         interface_group.setLayout(interface_layout)
 
+        comp_group = QGroupBox(_("Completion Widget"))
+        comp_label = QLabel(_("Decide what type of completion to use"))
+        comp_label.setWordWrap(True)
+        completers = [("plain", 0), ("droplist", 1), ("ncurses", 2)]

```

Comment:
Also, I don't like the names used in `completers`, given that those are shown in the dropdown box below too.

I know that those are the names used by IPython by I think they are too technical (like how many people in the Windows world knows what `ncurses` mean?).

What about these names?

* Graphical -> droplist
* Terminal -> ncurses
* Plain -> plain

I'm not sure about Terminal but I can't come up with a better name :-)